app/src/main/java/org/oppia/app/topic/TopicTab.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.app.topic
+
+enum class TopicTab(val position: Int) {
+  OVERVIEW(position = 0),
+  PLAY(position = 1),
+  TRAIN(position = 2),
+  REVIEW(position = 3);
+
+  var tabPosition: Int = position
```

Comment:
Why do we need this? ``position`` already contains the value, and this shouldn't ever change (so it shouldn't be a ``var``).

Reply:
Removed